client/modules/Registration/Registration.js

```diff
@@ -65,12 +77,24 @@ export class Registration extends Component {
         this.setState({ mobile: evt.target.value });
       }
       
-      handleSubmit = (evt) => {
+      handleSubmit(evt){
         if (!this.canBeSubmitted()) {
           evt.preventDefault();
           return;
         }
-        const { email, password, mobile, license } = this.state;
+        else{
+          evt.preventDefault();
+          http.client().post('/account/register', {
```

Comment:
we are asking for their license number but it does not get sent to the backend

Reply:
corrected

Reply:
latest commit creates license and image containers